tools/js-sdk-release-tools/src/test/changelog/testCases/patch.1.new.rlc.api.md

```diff
@@ -0,0 +1,38 @@
+# patch
+
+``` ts
+export interface ClustersGet {
+    a: string;
+}
+
+export interface ClusterGetOld {
```

Comment:
https://github.com/azure/autorest.typescript/blob/main/packages/typespec-test/test/faceai/generated/typespec-ts/review/ai-face-rest.api.md

Can we detect the changes of following overloads?

```ts
export interface AddPersonFace {
    get(options?: GetPersonFacesParameters): StreamableMethod<GetPersonFaces200Response | GetPersonFacesDefaultResponse>;
    post(options: AddPersonFaceParameters): StreamableMethod<AddPersonFace202Response | AddPersonFaceDefaultResponse>;
    post(options: AddPersonFaceFromUrlParameters): StreamableMethod<AddPersonFaceFromUrl202Response | AddPersonFaceFromUrlDefaultResponse>;
}
```

Can we detect any changes for inline expression?

```ts
export interface AddLargePersonGroupPersonFaceFromUrlBodyParam {
    // (undocumented)
    body: {
        url: string;
    };
}
```

Reply:
inline inline expression: yes 
overload in interface: no, let me support it

Reply:
I'm curious about these two:
``` ts
    post(options: AddPersonFaceParameters): StreamableMethod<AddPersonFace202Response | AddPersonFaceDefaultResponse>;
    post(options: AddPersonFaceFromUrlParameters): StreamableMethod<AddPersonFaceFromUrl202Response | AddPersonFaceFromUrlDefaultResponse>;
```

looks like `AddPersonFaceParameters` and `AddPersonFaceFromUrlParameters` the subtypes are the same, the only difference is the name, why we need to generate two different model in the same shape?

Reply:
> inline inline expression: yes overload in interface: no, let me support it

sorry, actually, this is already supported in old tool

Reply:
Ok, thanks